app/javascript/components/components/Header.js

```diff
@@ -17,47 +17,61 @@ class Header extends Component {
       sign_out_route,
       auction_new_route,
       my_auctions_route,
-      my_bids_route
-    } = this.props
+      my_bids_route,
+    } = this.props;
 
     return (
       <>
         <header>
           <div className="nav-bar">
             <NavLink to="/">
-              <img src={bidLogo} alt="Bid up logo" className="logo"/>
+              <img src={bidLogo} alt="Bid up logo" className="logo" />
             </NavLink>
           </div>
           <div className="nav-bar">
-            {logged_in &&
-              <NavLink to= "/auction_new_route" className="nav-link">Create Auction</NavLink>
-            }
-            {logged_in &&
-              <NavLink to= "/my_auctions_route" className="nav-link">My Auctions</NavLink>
-            }
-            {logged_in &&
-              <NavLink to= "/my_bids_route" className="nav-link">My Bids</NavLink>
-            }
+            {logged_in && (
+              <NavLink to="/auction_new_route" className="nav-link">
+                Create Auction
+              </NavLink>
+            )}
+            {logged_in && (
+              <NavLink to="/my_auctions_route" className="nav-link">
+                My Auctions
+              </NavLink>
+            )}
+            {logged_in && (
+              <NavLink to="/my_bids_route" className="nav-link">
+                My Bids
+              </NavLink>
+            )}
```

Comment:
You can group the nav links for logged in together as well as the logged out instead of setting up the conditional rendering for each particular link